src/main/java/com/google/devtools/build/lib/remote/RemoteOptions.java

```diff
@@ -65,6 +65,17 @@
   )
   public String remoteCache;
 
+  @Option(
+      name = "experimental_remote_cache_unix_socket",
+      defaultValue = "null",
+      documentationCategory = OptionDocumentationCategory.UNCATEGORIZED,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      converter = OptionsUtils.PathFragmentConverter.class,
+      help =
+          "Connect through a Unix domain socket instead of using the network."
```

Comment:
`experimental_remote_http_cache_unix_socket`

Please also mention in the help text that this flag only works for remote_http_cache and not for remote_cache (grpc).